Source/hydro/Castro_hydro_F.H

```diff
@@ -434,8 +434,8 @@ extern "C"
 
   void ca_mol_consup
```

Comment:
I'm confused by this interface change to ca_mol_consup; it doesn't look like you're passing shk in the C++ call to this function.

Reply:
I'm confused too.... I'll fix this